feather/server/src/main.rs

```diff
@@ -64,7 +70,11 @@ fn init_world_source(game: &mut Game, config: &Config) {
     // world otherwise. This is a placeholder:
     // we don't have proper world generation yet.
 
-    let seed = 42; // FIXME: load from the level file
+    let seed = config.world.seed.parse().unwrap_or_else(|_| {
+        let mut hasher = DefaultHasher::new();
```

Comment:
Should use the following hash algorithm `s[0]*31^(n-1) + s[1]*31^(n-2) + … + s[n-1]` which is javas `String.hashCode()`.

Reply:
Here's my implementation from #487: https://github.com/Iaiao/feather/blob/210fed57a4f54681c35d9d49956b1ced8f78c8fb/feather/server/src/main.rs#L95-L101

Reply:
> Here's my implementation from #487: https://github.com/Iaiao/feather/blob/210fed57a4f54681c35d9d49956b1ced8f78c8fb/feather/server/src/main.rs#L95-L101

Wouldn't it be useless for me to continue if you have already implemented this feature in your own fork?

Reply:
Maybe add a few tests, to make sure that the code behave as javas `String.hashCode()`.

Reply:
> > Here's my implementation from #487: https://github.com/Iaiao/feather/blob/210fed57a4f54681c35d9d49956b1ced8f78c8fb/feather/server/src/main.rs#L95-L101
> 
> Wouldn't it be useless for me to continue if you have already implemented this feature in your own fork?

My implementation doesn't have any tests and won't be merged soon (because it requires all vanilla commands to be implemented), so I think it wouldn't be useless to implement it in a separate pr.